distribution/src/conf/wrapper.conf

```diff
@@ -19,22 +19,22 @@ wrapper.tmp.path = ${jna_tmpdir}
 #********************************************************************
 # Java Application main class
 wrapper.java.app.mainclass=org.wso2.carbon.bootstrap.Bootstrap
-# Log Level for console output.  (See docs for log levels)
-wrapper.console.loglevel=INFO
+# Log Level for console output. (See docs for log levels)
+wrapper.console.loglevel=DEBUG
```

Comment:
Any reason to make the default as debug , shall we change back to INFO

Reply:
We have the loglevel as DEBUG in our minimal wrapper.conf configuration in the doc [1]. We can keep INFO by default and let the users update the loglevel accordingly. WDYT?

[1] https://docs.wso2.com/display/EI660/Running+the+Product+as+a+Windows+Service

Reply:
We always pack the wrapper conf as INFO and we have the same in kernel https://github.com/wso2/carbon-kernel/blob/4.6.x/distribution/kernel/carbon-home/bin/yajsw/wrapper.conf#L23.  It was added to EI 660 docs due to an issue in the packed wrapper conf in the default pack. Also IMO by default we need to provide in the INFO level and the user need to change if they need additional logs.

Reply:
Changed the default log level as INFO in 2b893df49e4d2501040005aa434da9ef8d8d24ca